07-lucid/06-traits.adoc

```diff
@@ -17,6 +17,16 @@ Using traits, you can
 3. Add methods to query builder instance for a given model.
 
 == Creating a Trait
+The `adonis` CLI provides a trait generator command.
```

Comment:
I use a little different language here and just to be consistent, can we change it to.


> You need to create a new file inside `app/Models/Traits` directory or use the following ace command to do it for you.